library/ssl_tls13_keys.c

```diff
@@ -321,10 +321,15 @@ int mbedtls_ssl_tls13_hkdf_expand_label(
                      const unsigned char *ctx, size_t ctx_len,
                      unsigned char *buf, size_t buf_len )
 {
-    const mbedtls_md_info_t *md_info;
     unsigned char hkdf_label[ SSL_TLS1_3_KEY_SCHEDULE_MAX_HKDF_LABEL_LEN ];
     size_t hkdf_label_len;
 
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
```

Comment:
Ok, this is something I should have made clear in the task description, sorry: TLS 1.3 is always PSA-based, contrary to 1.2 (where PSA has to remain optional for compatibility reasons until we can figure a way). So, while 1.2 (and other places of the library such as PK) have things like
```c
#if defined(MBEDTLS_USE_PSA_CRYPTO)
/* PSA-based code */
#else 
/* non-PSA alternative */
#endif 
```
TLS 1.3 will only have:
```c
/* PSA-based code */
```
except perhaps for the parts that are shared with TLS 1.2 (like record encrypt/decrypt, or tickets). But since HKDF is 1.3-only, this PR should not have to check for `MBEDTLS_USE_PSA_CRYPTO` anywhere, and it should just remove the non-PSA code.